### PR TITLE
Release Candidate 1.1

### PR DESCRIPTION
**Pull request by Neutrovertido**
## Features:
* Now supports modular commands!
* Added a persistent config.json file!
* Added moderation (kick & ban)!
* Added avatar display!
* Added a message eraser! (clean command)
* Added custom 8ball!
* Modular help command!

### CODE REVIEW

**Comment by Neutrovertido** [comment]
Merge or Squash will be done once testing ensures no issues and/or bugs are still present!
@Gasc1211 don't merge 'til then!

File: index.js [hunk 1]
```diff
@@ -66,4 +54,6 @@ bot.login(token);
 console.log("Login successful!");
```
Comment (gasc1211) [inline index.js:54]: We can do this in a single line.
```diff
- bot.login(token);
- console.log("Login successful!");
+ bot.login(token).then(() => console.log("Login Succesful!"));
```

File: commands/ban.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  name: "ban",
+  description: "Ban someone from the server.",
+  execute(message, args) {
+    const author = message.guild.member(message.author);
+    if (author.hasPermission("BAN_MEMBERS")) {
+      const target = message.mentions.users.first();
+      const reason = args;
```
Comment (gasc1211) [inline commands/ban.js:8]: I think we don't need reason variable on both ban and kick.
```diff
- const reason = args;
- const actualReason = reason.slice(1, reason.length).join(" ");
+ const actualReason = reason.slice(1, args.length).join(" ");
```

**Review by gasc1211 — CHANGES_REQUESTED** [review]
Code simplification and improvements.

**Pull request merged**